scripts/install/opt/seagate/cortx/motr/bin/motr_mini_prov.py

```diff
@@ -736,3 +736,28 @@ def lnet_self_ping(self):
        if ret != 0:
             return False
     return True
+
+def update_motr_hare_keys_for_all_nodes(self):
+    myhostname = self.server_node["hostname"]
+    nodes_info = Conf.get(self._index, 'server_node')
```

Comment:
Can we make read nodes into a generic code and use the same function at all places, this code is getting repeated in multiple places.

Reply:
Lets not clock this PR for making it generic and do this changes during refactoring of code.